src/Compilers/Core/Portable/DiagnosticAnalyzer/AnalyzerAssemblyLoader.cs

```diff
@@ -41,7 +40,15 @@ internal abstract partial class AnalyzerAssemblyLoader : IAnalyzerAssemblyLoader
         /// <remarks>
         /// Access must be guarded by <see cref="_guard"/>
         /// </remarks>
-        private readonly Dictionary<string, (AssemblyName? AssemblyName, string RealAssemblyPath, ImmutableHashSet<string> SatelliteCultureNames)?> _analyzerAssemblyInfoMap = new();
+        private readonly Dictionary<string, (AssemblyName? AssemblyName, string RealAssemblyPath)?> _analyzerAssemblyInfoMap = new();
+
+        /// <summary>
+        /// Set of analyzer dependencies original full paths to the satellite data calculated for that path
```

Comment:
IIUC, this says the key of the outer dict is the original full path, but it's not clear what the inner string value is, it just says "data calculated for that path".

Reply:
Yeah, this is confusing. Going to clean this up.

Reply:
OK, I think I cleaned things up (and fixed an issue) from when you last reviewed. @jjonescz  -- going to clear out your review status as it's changed a fair bit in the last iteration.